cdap-proto/src/main/java/io/cdap/cdap/proto/element/EntityType.java

```diff
@@ -51,7 +53,9 @@ public enum EntityType {
   INSTANCE(InstanceId.class),
   KERBEROSPRINCIPAL(KerberosPrincipalId.class),
   NAMESPACE(NamespaceId.class),
+  APPLICATIONREFERENCE(ApplicationReference.class),
```

Comment:
do we need to add these classes here? I'm not sure how the EntityType is being used, not see it in this PR.

Reply:
It's used in the `ApplicationReference` and `ProgramReference` class constructor. eg: ` super(namespace, EntityType.APPLICATIONREFERENCE);`